src/System.Net.Http.WinHttpHandler/tests/System.Net.Http.WinHttpHandler.Tests.csproj

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@mmitche I think we need both kinds of test projects, 1) the "unit test" kind that includes the source files directly compiled into the test assembly, and 2) the "functional test" kind that only links against the library.  For example, 1) allows us to test INTERNAL objects while 2) does not allow that.  

I would like to see consensus around this approach because both kinds of testing are important.
/cc @CIPop